### PR TITLE
SDK Caching for PercyScreenshot

### PR DESCRIPTION
**Pull request by Amit3200**
SDK Caching for PoA Sessions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +114,66 @@ If you have a previous Percy configuration file, migrate it to the newest versio
 ```sh-session
 $ percy config:migrate
 ```
+
+## Running Percy on Automate
```
Comment (itsjwala) [inline README.md:118]:
```suggestion
#### Running Percy on Automate
```

Comment (itsjwala) [reply README.md:118]: similar for other repos

**Review by itsjwala — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Amit3200) [reply README.md:118]: Issue is this is a different command and what I see is that migration config or installation of cli they are at `####` or `###` hence changing this to `####` will be weird.

Comment (itsjwala) [reply README.md:118]: ack, btw we're planning to move stuff to `docs.percy.io` to remove redundancy. though its fine for now.

**Pull request merged**